### PR TITLE
MD5: Ignore file extensions

### LINKED ISSUE #3974: MD5: Ignore filenames
### Description

Based on [feedback](https://github.com/duckduckgo/zeroclickinfo-goodies/issues/3901), one user was confused about the MD5 Instant Answer. They believed the md5 hash was for that file, and not the string "tails-i386-2.10.iso". 

screen shot 2017-03-02 at 4 00 58 pm

We need to update this IA so that it ignores file names. A suggested implementation would be to find file names with probable extensions using a RegEx.

## Get Started
- [ ] 1) Claim this issue by commenting below
- [ ] 2) Review our [Contributing Guide](https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/CONTRIBUTING.md)
- [ ] 3) [Set up your development environment](https://docs.duckduckhack.com/welcome/setup-dev-environment.html), and fork this repository
- [ ] 4) Create a Pull Request

## Resources
- Join [DuckDuckHack Slack](https://quackslack.herokuapp.com/) to ask questions
- Join the [DuckDuckHack Forum](https://forum.duckduckhack.com/) to discuss project planning and Instant Answer metrics
- Read the [DuckDuckHack Documentation](https://docs.duckduckhack.com/) for technical help

---

Instant Answer Page: http://duck.co/ia/view/md5

### PR DESCRIPTION
**Pull request by GPhantom**
## Description of new Instant Answer, or changes

Ignores the file extensions in the query.

## Related Issues and Discussions

Fixes #3974 

## People to notify

@pjhampton @gautamkrishnar 

---

Instant Answer Page: https://duck.co/ia/view/md5

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: lib/DDG/Goodie/MD5.pm [hunk 1]
```diff
@@ -23,6 +23,26 @@ handle remainder => sub {
...
+		gif|jpg|png|psd|pspimage|tga|thm|tif|tiff|
+		yuv|ai|eps|ps|svg|indd|pct|pdf|xlr|xls|xlsx|
+		accdb|db|dbf|mdb|pdb|sql|apk|app|bat|cgi|com|
+		exe|gadget|jar|wsf|dem|gam|nes|rom|sav|dwg|
+		dxf|gpx|kml|kmz|asp|aspx|cer|cfm|csr|css|
+		htm|html|js|jsp|php|rss|xhtml|crx|plugin|
+		fnt|fon|otf|ttf|cab|cpl|cur|deskthemepack|
+		dll|dmp|drv|icns|ico|lnk|sys|cfg|ini|prf|
+		hqx|mim|uue|7z|cbr|deb|gz|pkg|rar|rpm|sitx|
+		tar.gz|zip|zipx|bin|cue|dmg|iso|mdf|toast|
```
Comment (moollaza) [inline lib/DDG/Goodie/MD5.pm]: this `.` needs to be escaped

**Comment by moollaza** [comment]
@GPhantom @gautamkrishnar we need to back up changes we make with updated tests. Please add some example test the demonstrate this works.

File: lib/DDG/Goodie/MD5.pm (hunk 1 above)
Comment (GPhantom) [inline lib/DDG/Goodie/MD5.pm]: Done :+1:  #3981

**Pull request merged**